psychology_course_prereqs.csv

```diff
@@ -0,0 +1,43 @@
+Course,Prerequisite(s),,
```

Comment:
i would get rid of `(s)` and extra 2 commas.

Reply:
i'm not sure why that got added, but ill fix it